### ISSUE TITLE
--curve option and your pseudomask

### ISSUE THREAD

**Issue by Theodorenyu** [usr922/wseg#3]
Hi, I try to train the code and get performance curve like this:
```
19/60 background score: 0.190	mIoU: 59.990%
20/60 background score: 0.200	mIoU: 60.442%
21/60 background score: 0.210	mIoU: 60.832%
22/60 background score: 0.220	mIoU: 61.156%
23/60 background score: 0.230	mIoU: 61.306%
24/60 background score: 0.240	mIoU: 61.446%
25/60 background score: 0.250	mIoU: 61.529%
26/60 background score: 0.260	mIoU: 61.724%
27/60 background score: 0.270	mIoU: 61.746%
28/60 background score: 0.280	mIoU: 61.461%
29/60 background score: 0.290	mIoU: 61.201%
30/60 background score: 0.300	mIoU: 60.883%
31/60 background score: 0.310	mIoU: 60.706%
```
The best background score is 0.27 and I get 61.746% miou finally. This is close to your result, but I notice that this uses ground truth masks to choose an optimal background threshold. Can you give some insights about this?
Also, can you provide your processed pseudomasks so that others can use them to train segmentation models, thanks.

**Comment by usr922** [comment]
Hi, in fact, I was confused about this at first, but it almost became a common practice because the background is not supervised by image-level tags. You can view this issue (https://github.com/YudeWang/SEAM/issues/31) from SEAM for detailed explanation. Generally, existing works need to tune hyper-parameters, which inevitably requires a validation set with full mask annotations. So the curve option is also reasonable, which just follows previous works. However, I think this problem is far from being solved and it deserves further study. The pseudo masks will be uploaded later. :)

**Comment by Theodorenyu** [comment]
@usr922 Thanks a lot~
